### DISCUSSION TITLE
Discord Office Hours | .NET with Bruno | 9/6, 07:00 PM UTC / 12:00 PM PT

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#60] [Announcements]
Office hours starting soon!
[.NET with Bruno](https://discord.gg/T2eewYxR?event=1275538772208914442)

**Comment by elbruno** [comment]
See you in discord soon! 
Best 👍

**Comment by elbruno** [comment]
Shared links: 

- https://huggingface.co/microsoft/Phi-3.5-vision-instruct
- https://aka.ms/Phi-3CookBook
- https://github.com/microsoft/Phi-3CookBook/blob/main/md/04.Fine-tuning/FineTuning_Vision.md

Best
